script/_dns64

```diff
@@ -51,7 +51,7 @@ dns64_update_resolvconf()
     if [ $PLATFORM = ubuntu ]; then
         sudo resolvconf -u || true
     elif [ $PLATFORM = raspbian ]; then
-        sudo systemctl restart dhcpcd || true
+    sudo systemctl restart NetworkManager || true
```

Comment:
Shall we check if NETWORK_MANAGER is enabled? The `with/without` commands are designed to allow overridden by environment variables.
nit: indent.